### DISCUSSION TITLE
Can we perform all Tasks on Repl.it or some other online platform?

### DISCUSSION THREAD

**Discussion by askfriends** [spencerwooo/onedrive-vercel-index#144] [Q&A] [answered]
Hello;
First of all the GUI is so nice and looks much better than just plain, so thank you for that.

Im windows user, so can i perform all the tasks on Repl.it or any other online platform?

**Comment by spencerwooo** [comment]
why is using windows preventing you from deploying this project from your local device?

**Comment by spencerwooo** [comment, marked as answer]
im thinking that with the new version that's just released, all edits can be done in the browser, try it out ;)
